thprac/src/thprac/thprac_main.cpp

```diff
@@ -272,7 +271,7 @@ bool RunGameWithTHPrac(THGameSig& gameSig, std::wstring& name)
 
     auto result = (WriteTHPracSig(proc_info.hProcess) && THPrac::LoadSelf(proc_info.hProcess));
     if (!result)
-        TerminateThread(proc_info.hThread, -1);
+        TerminateThread(proc_info.hThread, ERROR_FUNCTION_FAILED);
```

Comment:
I think it'd be better to fix any warnings caused by `TerminateProcess(GetCurrentProcess(), ...)` lines by either changing it to `ExitProcess` or (if possible), returning back to WinMain, which then returns, right there in the same commit. I wanna get rid of all the  `TerminateProcess(GetCurrentProcess(), ...)` lines anyways cause that's jank

Reply:
I'm not sure if this comment was a duplicate or not, but the thread being terminated in this case seems to be the target game, not thprac itself.

Reply:
Oh, I must've misread and copy pasted the comment. Yeah in this case you'd call ExitThread

Reply:
Nevermind, I read that code completely wrong